resources/osl_shell_alias.rb

```diff
@@ -0,0 +1,37 @@
+resource_name :osl_shell_alias
+provides :osl_shell_alias
+unified_mode true
+
+default_action :add
+
+property :command, String, required: [:add]
+property :alias_name, String, name_property: true
+
+action :add do
+  file destination do
+    mode '0755'
+    content <<-EOF.gsub(/^ {6}/, '')
```

Comment:
Please use squiggly heredocs globally instead of `gsub`bing.

```suggestion
    content <<~EOF
```

Reply:
Didnt know about those, thanks!